### PR TITLE
adds base64 placeholder and srcset

### PR DESCRIPTION
**Pull request by andreialecu**

### CODE REVIEW

File: packages/fields-s3images/src/lib/s3.ts [hunk 1]
```diff
@@ -157,11 +161,33 @@ export const getDataFromStream = async (
     .promise();
   fileData.sizesMeta.lg = lgFileData;
 
+  if (config.sizes?.base64) {
+    const base64 = await imagePipeline
+      .clone()
+      .resize(config.sizes?.base64 || 10)
```
Comment (andreialecu) [inline packages/fields-s3images/src/lib/s3.ts:167]:
```suggestion
      .resize(config.sizes.base64)
```

File: packages/fields-s3images/src/lib/s3.ts [hunk 2]
```diff
@@ -157,11 +161,33 @@ export const getDataFromStream = async (
...
 
+  if (config.sizes?.base64) {
+    const base64 = await imagePipeline
+      .clone()
+      .resize(config.sizes.base64)
+      .toBuffer({ resolveWithObject: true });
+
+    const base64Data: ImagesData = {
+      id,
+      height: lgFile.info.height,
```
Comment (gautamsi) [inline packages/fields-s3images/src/lib/s3.ts:172]: - lgFile.info.height
+ base64.info.height

Comment (andreialecu) [inline packages/fields-s3images/src/lib/s3.ts:172]:
```suggestion
      height: base64.info.height,
```

File: packages/fields-s3images/src/lib/s3.ts [hunk 3]
```diff
@@ -157,11 +161,33 @@ export const getDataFromStream = async (
...
+    const base64 = await imagePipeline
+      .clone()
+      .resize(config.sizes.base64)
+      .toBuffer({ resolveWithObject: true });
+
+    const base64Data: ImagesData = {
+      id,
+      height: base64.info.height,
+      width: lgFile.info.width,
+      filesize: lgFile.info.size,
```
Comment (andreialecu) [inline packages/fields-s3images/src/lib/s3.ts:174]:
```suggestion
      height: base64.info.height,
      width: base64.info.width,
      filesize: base64.info.size,
```

File: packages/fields-s3images/src/lib/s3.ts [hunk 4]
```diff
@@ -157,11 +161,33 @@ export const getDataFromStream = async (
...
+      .resize(config.sizes.base64)
+      .toBuffer({ resolveWithObject: true });
+
+    const base64Data: ImagesData = {
+      id,
+      height: base64.info.height,
+      width: base64.info.width,
+      filesize: base64.info.size,
+      extension,
+      size: 'lg',
```
Comment (andreialecu) [inline packages/fields-s3images/src/lib/s3.ts:176]:
```suggestion
      size: 'base64',
```

File: packages/fields-s3images/src/lib/s3.ts [hunk 5]
```diff
@@ -157,11 +161,33 @@ export const getDataFromStream = async (
...
+      .clone()
+      .resize(config.sizes.base64)
+      .toBuffer({ resolveWithObject: true });
+
+    const base64Data: ImagesData = {
+      id,
+      height: base64.info.height,
+      width: base64.info.width,
+      filesize: base64.info.size,
+      extension,
```
Comment (andreialecu) [inline packages/fields-s3images/src/lib/s3.ts:175]:
```suggestion
      extension: 'png',
```

**Pull request merged**